src/libraries/System.Private.CoreLib/gen/System.Private.CoreLib.Generators.csproj

```diff
@@ -2,8 +2,6 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <LangVersion>10.0</LangVersion>
-    <Nullable>enable</Nullable>
-    <ILLinkTrimAssembly>false</ILLinkTrimAssembly>
```

Comment:
Are you sure this isn't needed anymore?

Reply:
Yes, `ILLinkTrimAssembly` isn't enabled for source generators: https://github.com/dotnet/runtime/blob/c96b44007e34fb43cb9ffb95a390410dc5b9db0d/src/libraries/Directory.Build.targets#L60.